src/coins/coins.ts

```diff
@@ -2804,6 +2804,32 @@ export const basicCoins: BasicCoin[] = [
     },
   },
 
+  {
+    key: CoinKey.XPL,
+    name: 'Plasma',
+    logoURI: 'https://s2.coinmarketcap.com/static/img/coins/64x64/36645.png',
+    verified: true,
+    chains: {
+      [ChainId.PLA]: {
+        address: '0x0000000000000000000000000000000000000000',
+        decimals: 18,
+      },
+    },
+  },
+
+  {
+    key: CoinKey.WXPL,
+    name: 'Wrapped Plasma',
+    logoURI: 'https://s2.coinmarketcap.com/static/img/coins/64x64/36645.png',
+    verified: true,
+    chains: {
+      [ChainId.PLA]: {
+        address: '', // to be updated
```

Comment:
Considering that we don't have this token address, let's start with alpha/beta types and make a main release only when we have all the details

Reply:
I have made it draft